cubic-server/entities/Entity.hpp

```diff
@@ -127,7 +127,7 @@ class Entity : public utility::SharedFromThis<Entity> {
     virtual void teleport(const Vector3<double> &pos);
 
     // Drop an item when necessary (death of the entity, broken block, ...)
-    // The dropped item is determined by the loot tables // TODO that's not true but should be
+    // @todo The dropped item is determined by the loot tables. Actually that's not true
```

Comment:
Remove the `Actually that's not true`, the `@todo` already specifies that it needs to be implemented